apps/feedahelp/src/components/ui/features/payment/payment.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import axios from "axios";
+import {
+  PayPalButtons,
+  FUNDING,
+} from "@paypal/react-paypal-js";
+
+interface PaymentProps{
+  value: string;
+  currency: string;
+  email?: string | undefined;
+}
+
+const Payment: React.FC<PaymentProps> = ({ value, currency, email }) => {
```

Comment:
This iwill create a huge merge conflicet , we either have rename to Paypal specific?